### DISCUSSION TITLE
VAR Guy 在2008年遴选出来的 top 50开源公司，现在是什么状况？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#94] [Announcements]
# VAR Guy 在2008年遴选出来的 top 50开源公司，现在是什么状况？ - 「开源之道」

https://opensourceway.community/posts/opensource/where_they_are_now_top_50_open_source_company/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

不是, 应该收集并对比中国开源公司的变化,

不过样本小很多, 现在俺有印象的就两家:

- APISIX
- PingCAP
